pycoal/cli/environment.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (C) 2017 COAL Developers
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth
+# Floor, Boston, MA 02110-1301, USA.
+# encoding: utf-8
+'''
+example_environment -- an example script which demonstrates COAL 
+environmental classification
+example_environment provides a CLI which demonstrates how the 
+COAL Environmental Classification API finds pixels in a mining
+classified image that are within a certain number of meters
+from features in a vector layer such as flow lines in the
+National Hydrography Dataset (NHD).
+More reading an this example can be seen at 
+https://capstone-coal.github.io/docs#usage
+@author:     COAL Developers
+@copyright:  2017 COAL Developers. All rights reserved.
+@license:    GNU General Public License version 2
+@contact:    coal-capstone@googlegroups.com
+'''
+
+import sys
+import os
+
+from argparse import ArgumentParser
+from argparse import RawDescriptionHelpFormatter
+
+import logging
+from subprocess import call
+import spectral
+import numpy
+from os.path import abspath, dirname, basename, splitext
+import pycoal
+import time
+sys.path.insert(0, '../pycoal')
+import mineral
+import mining
+import environment
+
+__all__ = []
+
+DEBUG = 1
+TESTRUN = 0
+PROFILE = 0
+
+from pycoal.write import Writer
+from pycoal.read_netcdf import NetCDFReader as Reader
```

Comment:
The code above has been copied from pycovjson, the ```Writer``` and ```NetCDFReader``` Objects do not exist in the pycoal project!